endpoints.go

```diff
@@ -32,7 +32,7 @@ var (
 	EndpointGatewayBot = EndpointGateway + "/bot"
 	EndpointWebhooks   = EndpointAPI + "webhooks/"
 
-	EndpointCDN             = "https://cdn.discordapp.com/"
+	EndpointCDN             = "https://cdn.discord.com/"
```

Comment:
The CDN URL (confusingly) hasn't changed.

Reply:
Done :+1: